Mathlib/Algebra/Module/LinearMap/Basic.lean

```diff
@@ -172,11 +178,16 @@ theorem map_smul_inv {σ' : S →+* R} [RingHomInvPair σ σ'] (c : S) (x : M) :
 #align semilinear_map_class.map_smul_inv SemilinearMapClass.map_smul_inv
 
 /-- Reinterpret an element of a type of semilinear maps as a semilinear map. -/
-abbrev semilinearMap : M →ₛₗ[σ] M₃ where
+@[coe]
```

Comment:
Does this work on bundled functions?

Reply:
I don't understand your question.

Reply:
@j-loreaux @Ruben-VandeVelde 
It seems I added the `@[coe]` tag, but I don't remember precisely why/when. (It even might have been there at some time.)
I will try to revert that change.